### PR TITLE
Update documentation

### LINKED ISSUE #294: Path names and management
Currently, path management is doe through the class `PathManager`. The path names are, however, too generic, such as `SAVE_PATH`, which can lead to clashes with other packages or variables.

**Solution**
Disambiguate paths by prepending `SIMPA_` to them. In addition, it should be allowed for the variables to be loaded directly from the environment, without having to define a `.env` file.

### PR DESCRIPTION
**Pull request by jnoelke**
**Please check the following before creating the pull request (PR):**
- [ ] Did you run automatic tests?
- [ ] Did you run manual tests?
- [x] Is the code provided in the PR still backwards compatible to previous SIMPA versions?
  
 **List any specific code review questions**
  
 **List any special testing requirements**
 
 **Additional context (e.g. papers, documentation, blog posts, ...)**
  Adds to the original pull request #296 
  
 **Provide issue / feature request fixed by this PR**
 
 Fixes #294

### CODE REVIEW

**Comment by kdreher** [comment]
This should probably not be its own PR but rather done everytime something in the documentation changes. One option could be to add a pre-commit hook that automatically builds the documentation

File: docs/source/introduction.md [hunk 1]
```diff
@@ -64,15 +64,19 @@ for further (and much better) guidance under:
 
 As a pipelining tool that serves as a communication layer between different numerical forward models and
 processing tools, SIMPA needs to be configured with the paths to these tools on your local hard drive.
-To this end, we have implemented the `PathManager` class that you can import to your project using
-`from simpa.utils import PathManager`. The PathManager looks for a `path_config.env` file (just like the
-one we provided in the `simpa_examples`) in the following places in this order:
+You have to a couple of options to define the required path variables.
```
Comment (kdreher) [inline docs/source/introduction.md:67]: Typo "to a"

**Review by kdreher — CHANGES_REQUESTED** [review]

**Comment by jnoelke** [comment]
As an alternative, it could also be implemented as a github action

**Review by kdreher — APPROVED** [review]
Looks good

**Pull request merged**